UPGRADE-5.0.md

```diff
@@ -11,11 +11,15 @@ EventDispatcher
 
  * The `TraceableEventDispatcherInterface` has been removed.
 
+FrameworkBundle
+---------------
+
+ * Using a `RouterInterface` that does not implement the `WarmableInterface` is not supported anymore.
```

Comment:
This must mention that it is about RouterCacheWarmer. This has nothing to do with the RouterInterface itself.